mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx

```diff
@@ -107,7 +107,7 @@ export const WayStatistic = (props: WayStatisticProps) => {
 
   const averageTimeForJob = Math.round(totalWayTime / allJobs.length);
 
-  const lastWeekDayReports = props.dayReports.filter((dayReport) => dayReport.createdAt > lastWeekDate);
+  const lastWeekDayReports = props.dayReports.filter((dayReport) => new Date(dayReport.createdAt.toDateString()) >= lastWeekDate);
```

Comment:
`new Date(dayReport.createdAt.toDateString())` looks a little hard to read. And also look like it is repeatable logic? Should we put it in some DateUtils helper and reuse everywhere?

Reply:
Done